### DISCUSSION TITLE
Large Dataset Clients v2 Process Check-in

### DISCUSSION THREAD

**Discussion by galen-mcandrew** [filecoin-project/notary-governance#327] [Archive]
When we approved issue #217 , we proposed running the experiment for 100PiB of DataCap. We are getting close to this total, when calculating the amount of DataCap granted to multisig notaries that have had at least one approved allocation.

**Checking in with the community, are there any strong concerns, arguments, or ideas for changes to the LDN process in the immediate future?**

We are increasing the tooling and dashboard tracking, which should continue to increase transparency and decrease notary lift. These improvements are ongoing, but do not require any fundamental changes to the process (client submits app, gov team approves, RKH create notary multisig, 2/23 notaries approve first allocation in app, bot tracks DataCap usage, once 75% sealed in deals bot requests subsequent allocation, two different notaries sign subsequent allocation).

**Comment by andrewxhill** [comment]
for sure, i documented some thinking on the initial allotment etc in this thread: https://github.com/filecoin-project/notary-governance/discussions/321

**Comment by andrewxhill** [comment]
this one also seems directly related: https://github.com/filecoin-project/notary-governance/discussions/235
ideally, we could increase quite a bit the automated allotments so that a typical customer journey might be:

1. discover filecoin.
2. get an automated verification that covers all testing and prototyping.
3. apply for LDN for all future production needs.

LDN feels much less cumbersome than the default process... so the underlying thought here is actually to flip and make default the LDN

**Comment by galen-mcandrew** [comment]
We had some great conversations and brainstorming on the current pain-points, which centered around allocation scale and frequency. While very important and those discussion will continue, this specific post was in regards to whether we are overall comfortable continuing the LDN process described in 217 past the 100PiB experiment stage.
